libcaca/riscv64.patch

```diff
@@ -1,14 +1,21 @@
+diff --git PKGBUILD PKGBUILD
+index 5b00085..dc7c30f 100644
 --- PKGBUILD
 +++ PKGBUILD
-@@ -41,3 +41,11 @@ package() {
+@@ -44,6 +44,9 @@ prepare() {
+   # https://github.com/cacalabs/libcaca/pull/66
+   git apply -3 ../libcaca-0.99.beta20-CVE-2022-0856.patch
+ 
++  # Disable LTO when checking for fsin/fcos/fldln2/...
++  git apply -3 ../libcaca-fix-instruction-detection.patch
++
+   autoreconf -fi
+ }
+ 
+@@ -67,3 +70,6 @@ package() {
    make DESTDIR="$pkgdir" install
-   install -Dm644 COPYING "$pkgdir/usr/share/licenses/$pkgname/COPYING"
+   install -Dm644 -t "$pkgdir/usr/share/licenses/$pkgname/" COPYING
  }
 +
-+source+=("$pkgname-fix-instruction-detection.patch::https://github.com/cacalabs/libcaca/pull/76.patch")
-+sha256sums+=('2d40a2c2654369e0583f9c26653b56d6b4306d76ad12c78478de489a8f91160b')
-+prepare() {
-+  cd $pkgname-$pkgver
-+  patch -Np1 -i ../$pkgname-fix-instruction-detection.patch
-+  autoreconf -fi
-+}
++source+=("libcaca-fix-instruction-detection.patch")
```

Comment:
Is the patch modified comparing with the upstream PR? If not, it is better to reference it by URL without storaging in the repo.